src/TesterInternal/StreamingTests/SMSStreamingTests.cs

```diff
@@ -35,30 +35,23 @@ public class SMSStreamingTests : UnitTestSiloHost
                         StartSecondary = false, 
                     };
 
-        private readonly SingleStreamTestRunner runner;
-        private readonly bool fireAndForgetDeliveryProperty;
+        private SingleStreamTestRunner runner;
+        private bool fireAndForgetDeliveryProperty;
 
-        public SMSStreamingTests()
-            : base(smsSiloOption, smsClientOptions)
+        public static TestingSiloHost CreateSiloHost()
         {
-            runner = new SingleStreamTestRunner(SingleStreamTestRunner.SMS_STREAM_PROVIDER_NAME);
-            fireAndForgetDeliveryProperty = ExtractFireAndForgetDeliveryProperty();
+            return new TestingSiloHost(
+                smsSiloOption,
+                //smsSiloOption_OnlyPrimary,
+                smsClientOptions);
         }
 
-        public SMSStreamingTests(int dummy)
-            : base(smsSiloOption_OnlyPrimary, smsClientOptions)
```

Comment:
there were some tests like this that had a non-default constructor, but no derived class, so not sure what was the purpose as MSTest would never run those tests. I kept some code commented out (such as the `//smsSiloOption_OnlyPrimary` in the CreateSiloHost method) but not sure about the usefulness of it. @sergeybykov  do you have any recommendation? If those tests are valid, then I could create a derived class that exercises that path.